contrib/examples/actions/mistral-customfilters-json-escape.yaml

```diff
@@ -0,0 +1,11 @@
+---
+description: Example for using the custom filter "json_escape"
+enabled: true
+entry_point: workflows/tests/mistral-json-escape.yaml
+name: mistral-customfilters-json-escape
```

Comment:
Please make naming consistent. Usually, I name the workflow and the file name the same.  For example, name the workflow `mistral-test-func-json-escape`, the meta file should be `mistral-test-func-json-escape.yaml` and the entry point should be `workflows/tests/mistral-test-func-json-escape.yaml`.